resources/templates/fly.toml

```diff
@@ -0,0 +1,50 @@
+# fly.toml app configuration file generated
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = "$APP_NAME$"
+kill_signal = "SIGINT"
+kill_timeout = 5
+primary_region = "ams"
+processes = []
+
+[build]
+[build.args]
+NODE_VERSION = "$NODE_VERSION"
```

Comment:
How about this line here at 14, it doesn't have ending $ like appname and phpversion